src/pages/patientView/PatientViewPage.tsx

```diff
@@ -36,6 +37,13 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
             );
         }
 
+        let mutationDiv: Element | null = document.getElementById('mutations_div_prototype');
```

Comment:
@alisman Ah my bad, I told @onursumer to do it like this. It's the same as we did before afaik. What is the new approach?